src/main/java/chess/Board.java

```diff
@@ -0,0 +1,63 @@
+package chess;
+
+import piece.Pawn;
+import piece.Piece;
+import piece.Pieces;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Board {
+
+    private static List<List<Piece>> BOARD = intialize();
```

Comment:
```suggestion
    private static List<List<Piece>> board = intialize();
```

상수가 아닌 객체는 대문자로 하지 않습니다.